src/web/mjs/engine/ComicInfoGenerator.mjs

```diff
@@ -0,0 +1,29 @@
+export default class ComicInfoGenerator {
+    static createComicInfoXML(series, title, pagesCount) {
+        series = this.escapeXML(series);
+        title = this.escapeXML(title);
+        return `<?xml version="1.0" encoding="utf-8"?>
+<ComicInfo xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <Title>${title}</Title>
+    <Series>${series}</Series>
+    <PageCount>${pagesCount}</PageCount>
+</ComicInfo>`;
+    }
+
+    static escapeXML(str) {
+        return str.replace(/[<>&'"]/g, function (c) {
+            switch (c) {
```

Comment:
Consider to use a predefined dictionary for mapping characters instead of `switch ... case`.